EDMarketConnector.py

```diff
@@ -351,13 +351,13 @@ def __init__(self, master: tk.Tk):  # noqa: C901, CCR001 # TODO - can possibly f
             self.menubar.add_cascade(menu=self.help_menu)
             if platform == 'win32':
                 # Must be added after at least one "real" menu entry
-                self.always_ontop = tk.BooleanVar(value=cast(int, config.get_int('always_ontop')))
-                self.system_menu = tk.Menu(self.menubar, name='system', tearoff=tk.FALSE)  # type: ignore # https://github.com/python/typeshed/issues/4658 # noqa: E501
+                self.always_ontop = tk.BooleanVar(value=config.get_int('always_ontop')))
+                self.system_menu=tk.Menu(self.menubar, name = 'system', tearoff = tk.FALSE)  # type: ignore # https://github.com/python/typeshed/issues/4658 # noqa: E501
```

Comment:
I thought the proper way was "foo=val" in arguments, and "foo = val" in assignments otherwise?  You're flipping a load to the opposite of that.

Reply:
I think theres an extra paren in there or something that caused the formatter to do that. Will fix when Im more awake tomorrow